utils/Rector/PassStrictParameterToFunctionParameterRector.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Utils\Rector;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Name;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Pass strict to function parameter on specific position argument when no value provided
+ */
+final class PassStrictParameterToFunctionParameterRector extends AbstractRector
+{
+	private const FUNCTION_WITH_ARG_POSITION = [
+		// position start from 0
+		'in_array' => 2,
+	];
```

Comment:
@paulbalandan I've refactored to allow register more functions to specific position to be filled with strict parameter in the above constant.

Reply:
Thank you for indulging my request that fast.